apps/docs/src/pages/components/section.mdx

```diff
@@ -33,12 +33,9 @@ export const getStaticProps = async ({ params }) => {
       defaultValue: false,
     },
   }}
-  children={
-    <div>
-      <p>My section example content</p>
```

Comment:
we should avoid `p` if we can, because nextra is currently overriding its styles